Feature/update model access doc page (#5385)
fishtown-analytics/docs.getdbt.com: website/docs/reference/resource-configs/access.md

<VersionBlock firstVersion="1.7">

> You can apply access modifiers in config files, including the `dbt_project.yml`, or to models one-by-one in `properties.yml`. Applying access configs to a subfolder modifies the default for all models in that subfolder, so make sure you intend for this behavior. When setting individual model access, a group or subfolder might contain a variety of access levels, so when you designate a model with `access: public` make sure you intend for this behavior.

Reviewer: great catch!

Author: thanks! can you approve/merge this then? @mirnawong1

Reviewer: thanks @cvlsk ! will review this for you now

There are multiple approaches to configuring access:

In `properties.yml`: 

> <File name='models/<properties_my_public_model>.yml'>

Reviewer: ```suggestion
<File name='models/properties_my_public_model.yml'>
```

```yml
version: 2

models:
  - name: my_public_model
    access: public # Older method, still supported
    
```
</File>
  
Or (but not both)

> <File name='models/<properties_my_public_model>.yml'>

Reviewer: ```suggestion
<File name='models/properties_my_public_model.yml'>
```

<VersionBlock firstVersion="1.7">

You can apply access modifiers in config files, including the `dbt_project.yml`, or to models one-by-one in `properties.yml`. Applying access configs to a subfolder modifies the default for all models in that subfolder, so make sure you intend for this behavior. When setting individual model access, a group or subfolder might contain a variety of access levels, so when you designate a model with `access: public` make sure you intend for this behavior.

There are multiple approaches to configuring access:

- In `properties.yml` using the older method: 

  <File name='models/properties_my_public_model.yml'>
  
  ```yml
  version: 2
  
  models:
    - name: my_public_model
      access: public # Older method, still supported
      
  ```
  </File>
  

> - Newer method (v1.7 or older). Use either the older method or the new method, but not both for the same model:

Reviewer: ```suggestion
- In `properties.yml` using the new method (for v1.7 or higher). Use either the older method or the new method, but not both for the same model:
```
